commands/psync.md

```diff
@@ -0,0 +1,13 @@
+Initiates a replication stream from the master.
+
+The `PSYNC` command is called by Redis replicas for initiating a replication
+stream from the master.
+
+For more information about replication in Redis please check the
+[replication page][tr].
+
+[tr]: /topics/replication
```

Comment:
This may be a style thing, but should links be absolute (e.g. with TLD)? I'm thinking if this used to power something not on redis.io (RedisInsight).

Reply:
That's the current style. Any adopters will need to work with/around/sideways :)